MAINTAINERS.rst

```diff
@@ -1,8 +1,14 @@
-Notes for maintainers cutting a release.
+Notes for maintainers.
 
 Pre-release test plan
 ---------------------
 
+0. If no release branch exists, fork the version numbers in the release branch
+   (vX.Y-branch) and the master branch. See "Cutting a release branch", below,
```

Comment:
>  fork the version numbers in the release branch (vX.Y-branch) and the master branch. 

I'm not sure exactly what you mean by this. I cannot fork from two branches.
Did you means something like:
> fork the version number from the master branch into a new release branch (vX.Y-branch)
   (vX.Y-branch).

Reply:
The two branches fork from each other.

Precondition: `vX.Y-branch` does not exist, `master` is at version `X.(Y-1)`.
Postcondition: `v.X.Y-branch` exists and is at version `vX.Y.0a1`, `master` is at version `vX.Y.99`.

Reply:
much clearer, could you write that instead ?

Reply:
>  could you write that instead ?

Sure. Added to the details section.